sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -251,6 +251,11 @@ class RocksDB(
     }
   }
 
+  @GuardedBy("acquireLock")
+  private val rocksDBFileMapping: RocksDBFileMapping = new RocksDBFileMapping()
```

Comment:
I guess the main idea of the approach is to make this variable not be accessed by the maintenance thread. Perhaps should have a comment.

Reply:
done